lib/ansible/modules/cloud/heroku/heroku_collaborator.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/python
+
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: heroku_collaborator
+short_description: "Add or delete app collaborators on Heroku"
+version_added: "2.6"
+description:
+  - Manages collaborators for Heroku apps.
+author:
+  - Marcel Arns <marcel.arns@moneymeets.com>
+requirements:
+  - heroku3
+options:
+  api_key:
+    description:
+      - Heroku API key
+    required: false
+  apps:
+    description:
+      - List of Heroku App names
+    required: true
+  suppress_invitation:
+    description:
+      - Suppress email invitation when creating collaborator
+    type: bool
+    required: false
+    default: "False"
+  user:
+    description:
+      - User ID or e-mail
+    required: true
+  state:
+    description:
+      - Create or remove the heroku collaborator
+    choices: ["present", "absent"]
+    required: false
+    default: "present"
+notes:
+  - C(HEROKU_API_KEY) and C(TF_VAR_HEROKU_API_KEY) env variable can be used instead setting c(api_key).
+  - If you use I(--check), you can also pass the I(-v) flag to see affected apps in msg.
+'''
+
+EXAMPLES = '''
+- heroku_collaborator:
+    api_key: YOUR_API_KEY
+    user: max.mustermann@example.com
+    apps: heroku-example-app
+    state: present
+
+- heroku_collaborator:
+    api_key: YOUR_API_KEY
+    user: '{{ item.user }}'
+    apps: '{{ item.apps | default(apps) }}'
+    suppress_invitation: '{{ item.suppress_invitation | default(suppress_invitation) }}'
+    state: '{{ item.state | default("present") }}'
+  with_items:
+    - { user: 'a.b@example.com' }
+    - { state: 'absent', user: 'b.c@example.com', suppress_invitation: false }
+    - { user: 'x.y@example.com', apps: ["heroku-example-app"] }
+'''
+
+RETURN = ''' # '''
```

Comment:
Return is not compulsory but will help end user to understand return value of module.

Reply:
The documentation says:

```
If your module doesn't return anything (apart from the standard returns), you can use ``RETURN = ''' # '''``.
```

Our module indeed doesn't return anything apart from the standard returns (`msg`). If you think that we should document something we would appreciate further hints.